contracts/RMRK/library/RMRKErrors.sol

```diff
@@ -2,59 +2,124 @@
 error RMRKBadConfig();
+/// Attempting to set the priorities with an array of length that doesn't match the resource length
 error RMRKBadPriorityListLength();
+/// Attempting to add a resource entry without a `Part`
 error RMRKBaseRequiredForParts();
+/// Attempting to transfer a soulbound (non-transferrable) token
 error RMRKCannotTransferSoulbound();
+/// Attempting to accept a child that has already been accepted
 error RMRKChildAlreadyExists();
+/// Attempting to interact with a child, using index that is higher than the number of children
 error RMRKChildIndexOutOfRange();
+/// Attempting to equip a `Part` that does not allow equipping to the desired address
 error RMRKEquippableEquipNotAllowedByBase();
+/// Attempting to use ID 0, which is not supported
+/// @dev The ID 0 in RMRK suite is reserved for empty values. Guarding against its use ensures the expected operation
 error RMRKIdZeroForbidden();
+/// Attempting to interact with a resource, using index greater than number of resources
 error RMRKIndexOutOfRange();
+/// Attempting to reclaim a child that can't be reclaimed
 error RMRKInvalidChildReclaim();
+/// Attempting to interact with an end-user accoutn when the contract account is expected
 error RMRKIsNotContract();
+/// Attempting to interact with a contract that had its operation paused
 error RMRKLocked();
+/// Attempting to add a pending child after the number of pending children has reached the limit (default limit is 128)
 error RMRKMaxPendingChildrenReached();
+/// Attempting to add a pending resource after the number of pending resources has reached the limit (default limit is
+///  128)
 error RMRKMaxPendingResourcesReached();
+/// Attempting to mint a number of tokens that would cause the total supply to be greater than maximum supply
 error RMRKMintOverMax();
+/// Attempting to mint a nested token to a smart contract that doesn't support nesting
 error RMRKMintToNonRMRKImplementer();
+/// Attempting to unnest a child before it is unequipped
 error RMRKMustUnequipFirst();
+/// Attempting to nest a child over the nesting limit (current limit is 100 levels of nesting)
 error RMRKNestingTooDeep();
+/// Attempting to nest the token to own descendant, which would create a loop and leave the looped tokens in limbo
 error RMRKNestingTransferToDescendant();
+/// Attempting to nest the token to a smart contract that doesn't support nesting
 error RMRKNestingTransferToNonRMRKNestingImplementer();
+/// Attempting to nest the token into itself
 error RMRKNestingTransferToSelf();
+/// Attempting to interact with a resource that can not be found
 error RMRKNoResourceMatchingId();
+/// Attempting to manage a resource without owning it or having been granted permission by the owner to do so
 error RMRKNotApprovedForResourcesOrOwner();
+/// Attempting to interact with a token or resource without being its owner or having been granted permission by the
+///  owner to do so
 error RMRKNotApprovedOrDirectOwner();
```

Comment:
Let's remove " resource" and add this:
/// @dev when a token is nested, only the direct owner (NFT parent) can mange it, approved addresses are not allowed in that case to ensure consistency.